ios/FluentUI.Demo/FluentUI.Demo/Info.plist

```diff
@@ -26,7 +26,7 @@
 		</dict>
 	</array>
 	<key>CFBundleVersion</key>
-	<string>137.8</string>
+	<string>137.9</string>
```

Comment:
nah this should be 137.8.9. 137.9 is already taken by main_0.9 branch https://github.com/microsoft/fluentui-apple/blob/main_0.9/ios/FluentUI.Demo/FluentUI.Demo/Info.plist

Reply:
That seems wrong, 0.9 should have moved to 138.